src/main/java/com/staxrt/tutorial/model/Post.java

```diff
@@ -0,0 +1,80 @@
+package com.staxrt.tutorial.model;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.springframework.data.annotation.CreatedBy;
+import org.springframework.data.annotation.LastModifiedBy;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name = "posts")
+@EntityListeners(AuditingEntityListener.class)
+public class Post {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+
+    @Column(name = "data", nullable = false)
+    private String data;
+
+    @Column(name = "privacy", nullable = false)
+    private String privacy;
```

Comment:
To powinien być jaki enum czy coś w tym stylu.

Reply:
Yup enum jest najprostszą metoda, chciałem porównywać stringi ale enum jest lepszym rozwiązaniem

Reply:
review